Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml

```diff
@@ -160,20 +206,51 @@ script:
     - contextPath: IP.RecordedFuture.LastSeen
       description: Risk last seen timestamp
       type: date
-    description: Returns threat intelligence information for an IP address in Recorded Future.
-  - name: file
-    arguments:
-    - name: file
+    - contextPath: IP.Tags
+      description: Tags that are associated with the Domain
+      type: String
+    - contextPath: IP.CommunityNotes.Note
+      description: '...'
```

Comment:
A summary note on the mentioning of the IP in a source monitored by Recorded Future.